Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -291,10 +314,9 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
     // MARK: -
 
     @discardableResult public func getTokenContainer(policy: AuthTokensCachePolicy) async throws -> TokenContainer {
+        Logger.subscription.debug("Get tokens \(policy.description, privacy: .public)")
         do {
-            Logger.subscription.debug("Get tokens \(policy.description, privacy: .public)")
-
-            let referenceCachedTokenContainer = try? await oAuthClient.getTokens(policy: .local) // the currently stored one
+            let referenceCachedTokenContainer = try? await oAuthClient.getTokens(policy: .local)
```

Comment:
What do you think about replacing `reference` with `current` in both  `referenceCachedTokenContainer` and `referenceCachedEntitlements`?

Reply:
fixed